review/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
+from django.urls import reverse
+# from PIL import Image
+
+
+class Rating(models.TextChoices):
+    ONE_STAR = ('1', 'One Star')
+    TWO_STARS = ('2', 'Two Stars')
+    THREE_STARS = ('3', 'Three Stars')
+    FOUR_STARS = ('4', 'Four Stars')
+    FIVE_STARS = ('5', 'Five Stars')
+    UNSPECIFIED = ('UN', 'Unspecified')
+
+
+class Review(models.Model):
+    # professional_id = models.ForeignKey(Professional, on_delete=models.SET_NULL)
+    # client_id = models.ForeignKey(Client, on_delete=models.SET_NULL)
+
+    # image = models.ImageField(default='default.jpg', upload_to='profile_pics')
+
+    description = models.TextField(blank=True)
+    rating = models.CharField(max_length=2, choices=Rating.choices, default=Rating.UNSPECIFIED)
+    date_posted = models.DateTimeField(default=timezone.now)
```

Comment:
In my opinion, it should automatically receive the date and time when the review was uploaded.
Because it is not recommended that he fill in another date